plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -1170,7 +1173,29 @@ else if (avroSchemaLiteral != null) {
         // Partition Projection specific properties
         tableProperties.putAll(partitionProjectionService.getPartitionProjectionHiveTableProperties(tableMetadata));
 
-        return tableProperties.buildOrThrow();
+        Map<String, String> baseProperties = tableProperties.buildOrThrow();
+
+        // Extra properties
+        Map<String, String> extraProperties = getExtraProperties(tableMetadata.getProperties())
+                .orElseGet(ImmutableMap::of);
+
+        Set<String> existingProperties = ImmutableSet.<String>builder()
```

Comment:
This is not accurate name, since these properties aren't "existing" yet (and STATS_PROPERTIES may or may not be set)

inline to avoid having to name this